### PR TITLE
fix: target click function was not working

### PR DESCRIPTION
**Pull request by Kashifalaliwala**

### CODE REVIEW

File: lib/showcase.dart [hunk 1]
```diff
@@ -212,6 +212,21 @@ class _ShowcaseState extends State<Showcase> with TickerProviderStateMixin {
     }
   }
 
+  _getOnTargetTap() {
+    if (widget.disposeOnTap == true) {
+      return widget.onTargetClick == null
+          ? () {
```
Comment (BirjuVachhani) [inline lib/showcase.dart:218]: No Need to wrap another `VoidCallback`.

File: lib/showcase.dart [hunk 2]
```diff
@@ -256,7 +271,7 @@ class _ShowcaseState extends State<Showcase> with TickerProviderStateMixin {
             _TargetWidget(
               offset: offset,
               size: size,
-              onTap: _nextIfAny,
+              onTap: _getOnTargetTap(),
```
Comment (BirjuVachhani) [inline lib/showcase.dart:274]: I don' think it is correct way to pass a function.

**Review by BirjuVachhani — CHANGES_REQUESTED** [review]

File: lib/showcase.dart (hunk 2 above)
Comment (Kashifalaliwala) [reply lib/showcase.dart]: Fixed

File: lib/showcase.dart (hunk 1 above)
Comment (Kashifalaliwala) [reply lib/showcase.dart]: Fixed.

**Review by BirjuVachhani — APPROVED** [review]
LGTM

**Pull request merged**